docs/cookbook.md

```diff
@@ -23,6 +23,8 @@
   - [UTC offset for a zoned event, as a number of seconds](#utc-offset-for-a-zoned-event-as-a-number-of-seconds)
   - [Offset between two time zones at an instant](#offset-between-two-time-zones-at-an-instant)
 - [Arithmetic](#arithmetic)
+  - [Unit-constrained duration between now and a past/future zoned event](#unit-constrained-duration-between-now-and-a-pastfuture-zoned-event)
```

Comment:
Is this an accident?

Reply:
No, it's a rebase race — because the TOC automatically updated and included the previous example which was merged before the TOC commit.

Reply:
No worries! I'll review and merge the TOC PR ASAP.